pyi.py

```diff
@@ -2162,6 +2163,9 @@ def _visit_function(self, node: ast.FunctionDef | ast.AsyncFunctionDef) -> None:
         with self.in_function.enabled():
             self.generic_visit(node)
 
+        if node.name != "__getattr__" and node.returns and _is_Incomplete(node.returns):
```

Comment:
`_is_Incomplete` accepts `ast.AST | None`, so I don't think we need this check here:

```suggestion
        if node.name != "__getattr__" and _is_Incomplete(node.returns):
```

Reply:
It's necessary to make mypy happy, as `self.error()` only accepts an `AST`.

Reply:
Oh, I see. Maybe we could make `_is_object` return `TypeGuard[ast.AST]`. But what you have is also fine.